index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+   <div class="nav"> 
+    <header> 
+        <a class=" logo" href="#"><h1>Welcome</h1></a> 
+    </header>
+    <nav>
+        <a href="#"><img id="MenuIcons" src="images/Icons/Menu.png" alt="togglebutton"></a>
+    </nav>
+    </div>
+    <main >
+        <section id="section-1">
+                <div class="Primary"><h2 class="Primary-1">Hey there. I’m Mariane</h2>
+                        <h2 id="Primary-2" >I’m a software developer</h2></div>
+                        <div class="supporting"> <p>I can help you build a product , feature or website Look through some of my work 
+                        and experience! If you like what you see and have a project you need coded,<br> 
+                        don’t hestiate to contact me.</p>
+                </div>
+                
+                <ul class=social>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/twitter.png" alt="Twitter icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/Linkedinicon.png" alt="Linkedin icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/medium.png" alt="medium icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/github.png" alt="github icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/angellist.png" alt="angellist icon"></a></div></li>
+                </ul>
+                
+               <div class="action"><a href="#">
+               <img src="images/Icons/Disabled.png" alt="Disabled"></a></div>
+        </section>
+
+            <div class="Primary"><h2 class="Primary-1">My Recent Works</h2><hr></div>
+        <section id="section-2" class="grid">
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
```

Comment:
- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 

```
 <ul class="cells">
      <li> <a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a>></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
</ul>
``` 
- [ ]  It would be nice if you fallow the same `padding` in the template for the technologies list. Check the deference below

Your design | the original
----------------| ---------------
![Screenshot from 2021-11-27 17-37-00](https://user-images.githubusercontent.com/65068771/143688700-3c57d2b0-6d35-4748-841d-714612b534cb.png)![Screenshot from 2021-11-27 17-46-51](https://user-images.githubusercontent.com/65068771/143688703-37bbfbe6-fb78-4582-b3db-547dfbbb9779.png) | ![Screenshot from 2021-11-27 18-13-41](https://user-images.githubusercontent.com/65068771/143688788-1b26ce26-4d9d-48f4-8294-46fc30a928d1.png)![Screenshot from 2021-11-27 18-14-07](https://user-images.githubusercontent.com/65068771/143688789-d4823657-ee1f-4638-a36a-cd1cbed37d64.png)